ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -112,25 +113,42 @@ export interface IconConfig {
 
 const displayRuntimeStatusIcon = (
   icon: IconConfig,
-  workspaceNamespace: string,
-  featureFlaggedStyling: boolean = false
+  workspaceNamespace: string
 ) => {
-  // We always want to show the thunderstorm icon.
-  // For most runtime statuses (Deleting and Unknown currently excepted), we will show a small
-  // overlay icon in the bottom right of the tab showing the runtime status.
-  return (
-    <FlexRow
-      style={{
+  const { enableGkeApp } = serverConfigStore.get().config;
```

Comment:
More out of curiosity, what does this do? If I am reading this right, it means to enable a Google Kubernetes Engine App. But what does that mean? Are all apps gke apps, or do they change somehow if this is true?

Reply:
Ahh ok. That's helpful context. Thank you.